### PR TITLE
add age as stats option

### PR DESCRIPTION
**Pull request by derkgort**
Hi,

I've added an option to add age as a stats option. this will help to see if there's old jobs in a queue

### CODE REVIEW

**Comment by derkgort** [comment]
I'm not a ruby expert. And did not really understand this issue:

bin/metrics-beanstalkd-tubes.rb:92:5: C: Use a guard clause instead of wrapping the code inside a conditional expression.
    if matches_filter?(:stats, 'age')
    ^^

What do i need to change?

**Comment by majormoses** [comment]
Sorry for the delayed response, what I believe it wants you to do is this:
```
output "#{config[:scheme]}.#{tube.name}.age", tube_age(tube) if matches_filter?(:stats, 'age')
```

It is a very subtle difference in ruby from a functional and performance perspective.

**Comment by majormoses** [comment]
Also I will need 2 things in order to merge this:
- CHANGELOG entry
- a testing artifact[1]

[1] A testing artifact can be an automated test or it can be a simple gist of you running the check and showing the output like this: https://github.com/sensu-plugins/sensu-plugins-aws/pull/230#issuecomment-321140616

**Comment by derkgort** [comment]
Hi,

The guard is accepted by Travis. Thanks!
And i've added the changelog + a gist with the change in effect. I've linked to that in the changelog.

**Comment by majormoses** [comment]
https://gist.github.com/derkgort/d05fba4368aed01c7908ee26366d7dda

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Added
+- Option to get the age of a item in the tubes. add age to the --stats
+  See https://gist.github.com/derkgort/d05fba4368aed01c7908ee26366d7dda for change in effect
```
Comment (majormoses) [inline CHANGELOG.md:9]: Sorry I was not clear this should just go in a comment in the PR. I merged and fixed this up.

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-beanstalk/versions/1.1.0

**Pull request merged**